lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -26,15 +26,15 @@
 
         <div class="team-capacity">
             <span class="sr"><%- gettext("Team capacity") %></span>
-            <span><%- membershipText %></span>
+            <span><%= HtmlUtils.ensureHtml(membershipText) %></span>
         </div>
 
         <% if (country) { %>
             <div class="team-country">
                 <span class="sr"><%- gettext("The country that team members primarily identify with.") %></span>
                 <i class="icon fa fa-globe fa-fw" aria-hidden="true"></i>
                 <span>
-                    <%- gettext(country) %>
+                    <%= HtmlUtils.ensureHtml(gettext(country)) %>
```

Comment:
This change doesn't seem necessary. ``<%- gettext(country) %>`` will correctly escape the text.